content/_index/overview/memoryinspector.md

```diff
@@ -0,0 +1,10 @@
++++
+title = "Memory Inspector"
+weight = 20
+
+[asset]
+  icon = "fas magnifying-glass"
+  url = "https://github.com/eclipse-cdt-cloud/vscode-memory-inspector"
++++
+
+VS Code extension containing a full featured memory inspector which uses the Debug Adapter Protocol
```

Comment:
Should we be in sync with this: https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/blob/main/package.json#L4
Could as well mean another update the package manifest rather than an update here